propagator/jaeger/lib/opentelemetry/propagator/jaeger.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require_relative './jaeger/text_map_extractor'
+require_relative './jaeger/text_map_injector'
+
+# OpenTelemetry is an open source observability framework, providing a
+# general-purpose API, SDK, and related tools required for the instrumentation
+# of cloud-native software, frameworks, and libraries.
+#
+# The OpenTelemetry module provides global accessors for telemetry objects.
+# See the documentation for the `opentelemetry-api` gem for details.
+module OpenTelemetry
+  # Namespace for OpenTelemetry propagator extension libraries
+  module Propagator
+    # Namespace for OpenTelemetry Jaeger propagation
+    module Jaeger
+      extend self
+
+      DEBUG_CONTEXT_KEY = Context.create_key('jaeger-debug-key')
+      private_constant :DEBUG_CONTEXT_KEY
+
+      TEXT_MAP_EXTRACTOR = TextMapExtractor.new
+      TEXT_MAP_INJECTOR = TextMapInjector.new
+
+      private_constant :TEXT_MAP_INJECTOR, :TEXT_MAP_EXTRACTOR
+
+      IDENTITY_KEY = 'uber-trace-id'
```

Comment:
This and the constants below can be private

Reply:
These are private now.